screens/aqi_screen/aqiPage.tsx

```diff
@@ -46,13 +49,13 @@ const Aqi: FC<AQIScreenProps> = (props) => {
 
 
         <Text style={styles.title}>AirOMeter</Text>
-        <Text style={styles.cityName}>Fresno</Text>
+        <Text style={styles.cityName}>{cityName}</Text>
 
 
         <View style={styles.aqiCircle}>
           <View style={styles.aqiTextContainer}>
             <Text style={styles.aqiTextTitle}>AQI</Text>
-            <Text style={styles.aqiTextNum}>35</Text>
+            <Text style={styles.aqiTextNum}>{aqiInfo.aqi}</Text>
```

Comment:
You may want to check if `aqiInfo` is null before trying to index into it.  You could prevent this by doing something like:

```
aqiInfo ? aqiInfo.aqi : "N/A"
```

Reply:
awesome! I'll add that in! thanks!